From e2570604277b0acebcfd941cc7306a57b2503393 Mon Sep 17 00:00:00 2001
From: Junping Du <junping_du@apache.org>
Date: Wed, 6 Apr 2016 08:32:35 -0700
Subject: [PATCH 1801/2748] =?UTF-8?q?MAPREDUCE-6670.=20TestJobListCache#test?=
 =?UTF-8?q?Eviction=20sometimes=20fails=20on=20Windows=20with=20timeout.=20C?=
 =?UTF-8?q?ontributed=20by=20Gergely=20Nov=C3=A1k.?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

(cherry picked from commit de96d7c88a42cd54bd88ce2de63122998e967efa)

Change-Id: Id5b230ce1519bc0358685a0f35cc4badf007be54
---
 .../hadoop/mapreduce/v2/hs/TestJobListCache.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
index ba5599c..8479936 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
@@ -45,7 +45,7 @@ public void testAddExisting() {
         cache.values().size());
   }
 
-  @Test (timeout = 1000)
+  @Test (timeout = 5000)
   public void testEviction() throws InterruptedException {
     int maxSize = 2;
     JobListCache cache = new JobListCache(maxSize, 1000);
-- 
1.7.9.5

